jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/media/impl/MediaServiceRestImpl.java

```diff
@@ -71,9 +72,22 @@ public Response updateMedia(final JpaMedia media, final AuthenticatedUser aUser)
     }
 
     @Override
-    public Response removeMedia(final String name, final AuthenticatedUser aUser) {
+    public Response removeMedia(final String name, String type, final AuthenticatedUser aUser) {
+        MediaType mediaType = null;
+        if (type == null) {
+            mediaType = MediaType.JDK;
+        }
+        if (type.contains("JDK")) {
```

Comment:
Don't test with a String here, we have the enum already defined with the string value so use that instead.
`MediaType.JDK.getDisplayName().equals(type)`